components/asana/sources/tag-added-to-task/tag-added-to-task.mjs

```diff
@@ -0,0 +1,97 @@
+import asana from "../../asana.app.mjs";
+
+export default {
+  key: "asana-tag-added-to-task",
+  type: "source",
+  name: "New Tag Added To Task (Instant)",
+  description: "Emit new event for each new tag added to a task.",
+  version: "0.1.0",
+  props: {
+    asana,
+    workspace: {
+      label: "Workspace",
+      description: "Gid of a workspace.",
+      type: "string",
+      propDefinition: [
+        asana,
+        "workspaces",
+      ],
+      optional: true,
+    },
+    project: {
+      label: "Project",
+      description: "Gid of a project.",
+      type: "string",
+      propDefinition: [
+        asana,
+        "projects",
+        (c) => ({
+          workspaces: c.worspace,
+        }),
+      ],
+      optional: true,
+    },
+    tasks: {
+      propDefinition: [
+        asana,
+        "tasks",
+        (c) => ({
+          projects: c.project,
+        }),
+      ],
+    },
+    db: "$.service.db",
+    http: {
+      type: "$.interface.http",
+      customResponse: true,
+    },
+  },
+
+  hooks: {
+    async activate() {
+      const response = await this.asana.createWebHook({
+        data: {
+          filters: [
+            {
+              action: "added",
+              resource_type: "task",
+            },
+          ],
+          resource: this.project,
+          target: this.http.endpoint,
+        },
+      });
+
+      this.db.set("hookId", response.gid);
+      this.db.set("tasks", this.tasks);
```

Comment:
Could you please create a `setter` and `getter` method for this db? We are avoiding call the `db.set` directly